### DISCUSSION TITLE
More options for node display

### DISCUSSION THREAD

**Discussion by ladlon** [opentoonz/opentoonz#6740] [Ideas]
Hello...  So, I'd love to see some more options for the appearance of nodes... notably the font size, and an option to make nodes less tall (by making them more wide).

Perhaps the FX name could be optionally omitted (since you can give nodes custom names anyway), and maybe the info within the node box could be organized in a single line, with the input/output on the outer edges, and all the other items between them on a single line.

With a less tall node, they would line up nicer to the multi-inputs of something like an OverFX node, rather than having to have the various upstream nodes (going into its input bay) have to be fanned out.

And the font size would allow for more of the title to be displayed, as its currently getting pretty brutally truncated.

Certainly not a code red, but something I'd like to see, if poosible.  The ability to group nodes certainly helps with clutter, but often I find the height difference between nodes vs the distance betwen input bays of things like the OverFX node, can cause a lot of fanning and prevent clean alignment for easier path tracing and input ID'ing.

**Comment by RodneyBaker** [comment]
Some of the concerns of this request have been resolved.
For instance, the Schematic has the ability to snap nodes into place for better organization.
Recommend reviewing other parts of the request an updating details where necessary.
